CriticalMass/MapViewController.swift

```diff
@@ -221,10 +242,26 @@ extension MapViewController: MKMapViewDelegate {
             return nil
         }
 
-        guard let matchingController = annotationController.first(where: { type(of: annotation) == $0.annotationType }) else {
+        guard let matchingController = annotationControllers.first(where: { type(of: annotation) == $0.annotationType }) else {
             return nil
         }
 
+        // TODO: Remove workaround when target > iOS10 since it does not seem to work with the MKMapView+Register extension
```

Comment:
Yes it is the whole if block since it should work with 
```swift
return mapView.dequeueReusableAnnotationView(ofType: matchingController.annotationViewType, with: annotation)
```
but it does not